src/index.tsx

```diff
@@ -8,9 +8,9 @@ const root = ReactDOM.createRoot(
   document.getElementById('root') as HTMLElement
 );
 root.render(
-  <React.StrictMode>
-    <App />
-  </React.StrictMode>
+  // <React.StrictMode>
```

Comment:
Why drop strictmode?

Reply:
Dealing with the too many requests error from Blockfrost